spec/decisions/back_decision_tree_spec.rb

```diff
@@ -9,72 +9,103 @@
   let(:work_items) { [] }
   let(:disbursements) { [] }
 
-  it_behaves_like 'a generic decision', from: 'nsm/steps/firm_details', goto: { action: :show, controller: 'nsm/steps/start_page' }
-  it_behaves_like 'a generic decision', from: 'nsm/steps/contact_details', goto: { action: :edit, controller: 'nsm/steps/firm_details' }
+  it_behaves_like 'a generic decision', from: 'nsm/steps/firm_details',
+goto: { action: :show, controller: 'nsm/steps/start_page' }
```

Comment:
indentation looks strange, but might just be new rubocop rules strange

Reply:
Thought the same but it's a macro so Rubocop can't easily guess the indentation. Could add more line breaks to help it

Reply:
yeah that indeed looks odd, I edited it to make the indentation more readable and rubocop didn't say anything about it though